letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -643,6 +656,16 @@ def _update_ssl_vhosts_addrs(self, vh_path):
 
         return ssl_addrs
 
+    def _remove_existing_ssl_directives(self, vh_path):
+        for directive in ["SSLCertificateKeyFile", "SSLCertificateChainFile",
+                          "SSLCACertificatePath", "SSLCertificateFile"]:
```

Comment:
That's kind of subtle because of the insane cross-entanglement between client cert verification and chain building as documented [here](https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile).  "Hopefully" nobody has `SSLCACertificatePath` if they don't use client certs, but in any case leaving that directive around shouldn't be a cause of breakage.  I hope.